boards/build.rs

```diff
@@ -24,6 +24,9 @@ fn main() {
         panic!("Boards must provide a `layout.ld` link script file");
     }
 
+    println!("cargo:rustc-link-arg=-L{}", std::env!("CARGO_MANIFEST_DIR"));
+    println!("cargo:rustc-link-arg=-T{}", LINKER_SCRIPT);
```

Comment:
```suggestion
    // `-Tlayout.ld`: Use the linker script `layout.ld` all boards must provide.
    println!("cargo:rustc-link-arg=-T{}", LINKER_SCRIPT);
```

This comment got lost; though you could make the case it isn't the most useful of comments.